classes/renderers/submission_renderer.php

```diff
@@ -0,0 +1 @@
+
```

Comment:
Also empty

Reply:
I opened a ticket CTP-3714 to remove this and the other empty file